documentation/specs/proposed/BuildCheck.md

```diff
@@ -0,0 +1,405 @@
+
+# BuildCheck - Design Spec
+
+AKA MSBuild Analyzers
```

Comment:
Meta-comment: as a .NET dev, the term "analyzers" suggests to me that this is static analysis on the XML language itself, however that's not the case. Consider coming up with a different name to avoid overloading the term. That would also keep the name free for any future static analysis of MSBuild's language.

Reply:
This is a good point showing that we might need to have a short section on the actual OM and plans for that. As static analysis should actually be a subset - not a distinct set - of our analyses.

In any case - 'Analyzers' is mentioned as that's how we used to refere to the feature - so the reader know it's still same thing

Reply:
Yeah responding to similar feed back we renamed and now the different name is "BuildCheck".